libraries/AP_Scripting/generator/src/main.c

```diff
@@ -528,6 +550,8 @@ int parse_type(struct type *type, const uint32_t restrictions, enum range_check_
     string_copy(&(type->data.enum_name), data_type);
   } else if (type->type == TYPE_LITERAL) {
       string_copy(&(type->data.literal), data_type);
+  } else if (strcmp(data_type, keyword_alias) == 0) {
+    error(ERROR_USERDATA, "Cant add alias to unknown function");
```

Comment:
tiny whitespace issue here I think.  just indent a couple of characters.

Reply:
It's actually not a whitespace issue here, the whole file was 2 space indentation. Honestly the 4 spaces up on line 525 is the bigger change.

Reply:
I have fixed the indentation on the above line.